File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int> s;
        
        for(int i = 0; i < nums.size(); i++) {
            nums[s.size()] = nums[i];
            s.insert(nums[i]);
        }

        return s.size();
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        ptr = 0
        
        for i in range(1, len(nums)):
            if nums[i] == nums[ptr]:
                continue
            else:
                ptr += 1
                nums[ptr] = nums[i]
        
        return ptr+1
